igvc_platform/src/imu/SerialInterface.h

```diff
@@ -46,47 +46,47 @@ class SerialInterface
    */
   bool isConnected()
   {
-    return connected;
+    return connected_;
   }
 
   /**
-   * Destructor for the interface
+   * Destructor
    */
   ~SerialInterface()
   {
-    if (connection_port != NULL)
+    if (connection_port_ != NULL)
     {
-      if (connection_port->isOpen())
+      if (connection_port_->isOpen())
       {
         ROS_INFO_STREAM(this->log_zone_ << " Closing the Serial Port");
-        connection_port->close();
-        connected = false;
+        connection_port_->close();
+        connected_ = false;
       }
     }
   }
 
   /**
    * Connects to the serial port.
    */
-  void SerialConnect()
+  void serialConnect()
   {
     try
     {
-      connection_port.reset(new Serial(port_, (uint32_t)baud_, Timeout::simpleTimeout(60000)));
+      connection_port_.reset(new Serial(port_, (uint32_t)baud_, Timeout::simpleTimeout(60000)));
```

Comment:
Sorry for more changes, but I think going for
```c++
const auto uint_baud = static_cast<uint32_t>(baud_);
const auto timeout = Timeout::simpleTimeout(60000);
connection_port_ = std::make_unique<Serial>(port_, baud_, timeout);
```
 a bit better.

Also, what are the units on that timeout?